lib/features/sks-menu/data/models/dish_category_enum.dart

```diff
@@ -0,0 +1,36 @@
+import "package:flutter/cupertino.dart";
+import "package:freezed_annotation/freezed_annotation.dart";
+
+import "../../../../utils/context_extensions.dart";
+
+@JsonEnum(fieldRename: FieldRename.screamingSnake)
+enum DishCategory {
+  salad,
+  soup,
+  vegetarianDish,
+  meatDish,
+  sideDish,
+  drink;
+
+  @override
+  String toString() => name;
+}
+
+extension GetLocalizedNameX on DishCategory {
+  String getLocalizedName(BuildContext context) {
+    switch (this) {
```

Comment:
So in this case this will be only matter of formatting, but you can make it few lines shorter with the new function style switch case: https://www.christianfindlay.com/blog/dart-switch-expressions

```dart
return switch(this) {
   DishCategory.salad => context..... , 
   DishCategory.soup => context....., 
//...... 
};
```

Reply:
I like the solution you proposed!